spec/index.bs

```diff
@@ -6691,7 +6709,354 @@ An <dfn dfn>Extent3D</dfn> is a {{GPUExtent3D}}.
         or the third item of the sequence.
 </div>
 
-# Temporary usages of non-exported dfns # {#temp-dfn-usages}
+# Appendices # {#appendices}
+
+## Texture Format Capabilities ## {#texture-format-caps}
+
+Issue: add multisampling to these tables
+
+### Plain color formats ### {#plain-color-formats}
+
+All plain color formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/COPY_DST}}, and {{GPUTextureUsage/SAMPLED}} usage.
+
+Only formats with {{GPUTextureComponentType/float|GPUTextureComponentType."float"}} can be blended.
+
+The {{GPUTextureUsage/STORAGE|GPUTextureUsage.STORAGE}} column specifies the support for {{GPUTextureUsage/STORAGE}} usage in the core API. If {{GPUExtensionName/"extended-storage-formats"}} extension is enabled on the device, all of the formats in this table can be created with {{GPUTextureUsage/STORAGE}} flag.
+
+The "Filter" column specifies whether textures of this format can be sampled in shaders with a sampler having {{GPUFilterMode/linear}} filtering.
+
+<table class='data'>
+    <tr>
+        <th>Format
+        <th>Component
```

Comment:
optional nit: Instead of this column, we could just say "unorm, snorm, float, and ufloat formats have component type float; uint formats have component type uint; sint formats have component type sint". Maybe in table form, with float/norm/int colors. See PR.